src/Util/ValueConverter.php

```diff
@@ -70,12 +70,6 @@ public static function convert($value, TypeInterface $type): ?ValueNodeInterface
             return self::convertListType($value, $type);
         }
 
-        // Populate the fields of the input object by creating ASTs from each value
-        // in the PHP object according to the fields in the input type.
-        if ($type instanceof InputObjectType) {
-            return self::convertInputObjectType($value, $type);
-        }
-
```

Comment:
@Jalle19 Why was this removed?

Reply:
Unused, hard to test, didn't work correctly